app/src/main/proto/vss.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package org.vss;
+
+message GetObjectRequest {
+  /*
+    All APIs operate within a single storeId.
+    StoreId is keyspace identifier.
+    Ref: https://en.wikipedia.org/wiki/Keyspace_(distributed_data_store)
+    It is up to clients to use single or multiple store's for their usage.
+   */
+  string storeId = 1;
+  string key = 2;
+}
+message GetObjectResponse {
+  KeyValue value = 2;
+}
+
+message PutObjectRequest {
+  string storeId = 1;
+  /*
+    If present, write will only succeed if current DB globalVersion against a storeId is
+    same as in request.
+    Client is expected to send their current global version at hand.
```

Comment:
"at hand" is a little vague. We should spell out what this means.

Reply:
"Clients are expected to store version against every key locally. 
When initiating a PutRequest, request should contain current version for that key."
Let me know if this sounds better or we need to add more context.

Reply:
Not sure I follow what is meant by "store version against every key locally". This comment is on `globalVersion`, and my understanding is there is one global version for a store.